packages/modules/src/modules/banners/common/choiceCard/Button.tsx

```diff
@@ -1,100 +0,0 @@
-import React from 'react';
```

Comment:
I've moved the bits we still need from this file into `ChoiceCardsSupportCta.tsx`